torch/optim/rmsprop.py

```diff
@@ -125,20 +127,22 @@ def step(self, closure=None):
 
                 # State initialization
                 if len(state) == 0:
-                    state['step'] = 0
+                    state['step'] = torch.tensor(0)
```

Comment:
I am not sure this is ok?
In particular when working on GPU, it makes a big difference if this is a Tensor or a scalar.

Reply:
I think earlier in some other PR having this as tensor was a problem... (as it gets put on another device if checkpoint is loaded directly onto GPU, then incrementing a GPU tensor becomes slow) I remember it being reverted somewhere...